libraries/AP_HAL_ChibiOS/hwdef/F4BY_H743/hwdef.dat

```diff
@@ -0,0 +1,189 @@
+# hw definition file for processing by chibios_hwdef.py
+# for F4BY H743 v3.x.x board description https://f4by.com/en/?doc/fc_f4by_v3.0.1_h743
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID defined in tools board_types.txt as 
+# AP_HW_F4BY_H743
+APJ_BOARD_ID AP_HW_F4BY_H743
+
+FLASH_SIZE_KB 2048
+FLASH_RESERVE_START_KB 128
+# env OPTIMIZE -O2
+
+# crystal frequency
+OSCILLATOR_HZ 8000000
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 5
+
+
+# this is the pin that senses USB being connected. It is an input pin
+# setup as OPENDRAIN
+PA9 VBUS INPUT OPENDRAIN
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+PA13 JTMS-SWDIO SWD
+PA14 JTCK-SWCLK SWD
+
+# use UART for stdout, so no STDOUT_SERIAL
+# STDOUT_SERIAL SD5
+# STDOUT_BAUDRATE 57600
+
+
+SERIAL_ORDER OTG1 USART2 USART1 USART3 UART4 UART5 
+DMA_PRIORITY S*
+
+
+# CAN bus
+PD0 CAN1_RX CAN1
+PD1 CAN1_TX CAN1
+
+# UART1 
+PB6 USART1_TX USART1 # TIM4_CH1 TIM16_CH1N
+PB7 USART1_RX USART1 # TIM4_CH2 TIM17_CH1N 
+
+# USART2 
+PD5 USART2_TX USART2 NODMA
```

Comment:
if you get rid of the NODMA you will have a UART for use with networking that has full DMA , You could not add bdshot to the last group to avoid and sharing issues (UART1 will share with TIM8 otherwise)